pkg/virt-operator/install-strategy/strategy.go

```diff
@@ -73,7 +59,7 @@ type InstallStrategy struct {
 	deployments []*appsv1.Deployment
 	daemonSets  []*appsv1.DaemonSet
 
-	customSCCPrivileges []*customSCCPrivilegedAccounts
```

Comment:
unfortunately, I think we have to leave this struct and logic around patching `privileged` scc in for now. Otherwise we break updates.

So the logic that updates the `privileged` SCC needs to stay, but the new logic that generates the InstallStrategy should just not set anything on this array and use the new `sccs` slice instead.

Reply:
would a simplified logic be enough? Just removing users from the privileged SCC? We don't have a history of changes here yet, we know what we did to the privileged SCC.

Reply:
I think we can even get rid of the struct. loadInstallStrategyFromBytes() will just ignore the old config. And we just remove users from the privileged SCC when creating/updating/deleting the new SCCs. Does that make sense?

Reply:
how do we know what users to remove without this struct?

Reply:
also, we need the add logic in the event that we have to restore a degraded install to the previously working version.

Reply:
> how do we know what users to remove without this struct?
we just know which users were ever added by our code, so we could hardcode them.
but yes, restoring (or even just installing) N-1 is a good reason why we need to keep the struct and logic for now. Too bad.